.github/workflows/api-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: API-Tests
+on:
+  pull_request:
+    types: [ready_for_review]
+
+jobs:
+  api-tests:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # chagne this to use docker compose rather than docker-compose 
+      # this step is only required locally when using act since the ubuntu image
+      # does not come with docker-compose installed, however the alternative to the reinstall
+      # here is you'd have to install a custom docker image locally which includes docker-compose
+      - name: Install docker-compose
```

Comment:
Ideally either comment this out or put a check to see if docker compose is available before downloading and installing.  It doesn't seem to cause an issue though

Reply:
updated comment and commented out